src/integration/pushResult.js

```diff
@@ -73,13 +106,34 @@ function addCommit(src, msg){
 
 //pushing to remote repo
 function push(branch, src){
-
   return (cb) => {
-
     git.push('origin', branch, {cwd: src}, (err) => {
       cb(err);
     });
   };
 }
 
 module.exports = pushResult;
+
+/**
+ * Function resposible for moving files between two directories in order to backup them or restore them
+ * @param {Boolean} [from] - responsible for choosing the src folder: `${item}/*` or `./tmp/backup/${path.normalize(item)}/*`
+ * @param {Boolean} [to] - responsible for choosing the destination folder: `${item}/` or `./tmp/backup/${path.normalize(item)}/`
+ * @param {String} [tempLocation] - indicates folder with cloned repositories
+ * @param {Function} [cb] - callback for asynchronous operation
+*/
+
+function _backup(from, to, tempLocation, cb){
+  const data = fs.readFileSync(`./${tempLocation}/notClonedRepositories.json`, 'utf-8');
```

Comment:
getFilesFromLocations is a function that will read path or array of paths and return dirs/files in them

We need to read the content of the file. It cannot be used here.